client/transaction-pool/src/revalidation.rs

```diff
@@ -245,6 +245,16 @@ impl<Api: ChainApi> RevalidationWorker<Api> {
 						Some(worker_payload) => {
 							this.best_block = worker_payload.at;
 							this.push(worker_payload);
+
+							if this.members.len() > 0 {
+								log::debug!(
+									target: "txpool",
+									"Updated revalidation queue at {}. Transactions: {:?}",
```

Comment:
Does it print the entire transaction content or just hashes? Might be worth to format the transactions so that each one is in it's own line for readability if we don't expect this collection to be super big.

Reply:
Just hashes. I think `{hash1, hash2}` (debug impl of HashSet) is quite readable, we use similar in block production. The data will be used mostly for hash search anyway,  not for reading the entire list.

Reply:
Yeah, that sounds good then!